src/components/pages/Table/ContinentSelect.jsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import { connect } from 'react-redux';
-import { setContinentFilter } from '../../../state/actions';
+import { setContinentFilter } from '../../../state/actionCreators';
```

Comment:
The file renaming is very helpful in understanding what exactly it is you're working with.